api/Element.json

```diff
@@ -611,7 +611,7 @@
               "version_added": "43"
             },
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
This is wrong `document.body.className` works in Edge 12. But it's on `HTMLElement.prototype` and not `Element`. Moved to `Element` in Edge 12 it seems.

Reply:
This is now tracked in https://github.com/mdn/browser-compat-data/issues/3463